shared/people/container.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import People from './'
+import * as PeopleGen from '../actions/people-gen'
+import * as Types from '../constants/types/people'
+import * as Tabs from '../constants/tabs'
+import * as SettingsTabs from '../constants/settings'
+import {createGetMyProfile} from '../actions/tracker-gen'
+import {connect} from 'react-redux'
+import {type TypedState} from '../util/container'
+import {createSearchSuggestions} from '../actions/search-gen'
+import {navigateAppend, switchTo, navigateTo} from '../actions/route-tree'
+import {createShowUserProfile} from '../actions/profile-gen'
+import openURL from '../util/open-url'
+import {isMobile} from '../constants/platform'
+import {getPeopleDataWaitingKey} from '../constants/people'
+// import flags from '../util/feature-flags'
+
+const INSTALL_LINK_URL = 'https://keybase.io/download'
+
+const mapStateToProps = (state: TypedState) => ({
+  newItems: state.people.newItems,
+  oldItems: state.people.oldItems,
+  followSuggestions: state.people.followSuggestions,
+  myUsername: state.config.username,
+  waiting: !!state.waiting.get(getPeopleDataWaitingKey),
+})
+
+const onSkipTodo = (type: Types.TodoType, dispatch: Dispatch) => () =>
+  dispatch(PeopleGen.createSkipTodo({type}))
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  getData: () => dispatch(PeopleGen.createGetPeopleData({markViewed: true, numFollowSuggestionsWanted: 10})),
+  todoDispatch: {
+    bio: {
```

Comment:
lets discuss this. likely better to just move this into those components and have them be connected instead of it all being gathered here